Formula/insighttoolkit.rb

```diff
@@ -0,0 +1,111 @@
+class Insighttoolkit < Formula
+  desc "ITK is a toolkit for performing registration and segmentation"
+  homepage "https://www.itk.org"
+  url "https://downloads.sourceforge.net/project/itk/itk/4.12/InsightToolkit-4.12.2.tar.gz"
+  sha256 "8ba2183111723c4405af80724c60d07484c2bff8120ee4cef543bb726d07855e"
+  head "git://itk.org/ITK.git"
```

Comment:
we'll need to do without a head spec if there's no https

Reply:
Will update to https://itk.org/ITK.git